app/src/main/java/com/simongellis/vvb/game/GamePreferences.kt

```diff
@@ -66,11 +71,14 @@ class GamePreferences(context: Context) {
     val stereoSettings
         get() = StereoRenderer.Settings(screenZoom, aspectRatio.ordinal, verticalOffset, color)
 
+    val leiaSettings
+        get() = LeiaRenderer.Settings(screenZoom, aspectRatio.ordinal, verticalOffset, color, colorBG)
+
     init {
 
         val prefs = PreferenceManager.getDefaultSharedPreferences(context)
 
-        videoMode = VideoMode.valueOf(prefs.getString("video_mode", VideoMode.ANAGLYPH.name)!!)
+        videoMode = VideoMode.valueOf(prefs.getString("video_mode", VideoMode.LEIA.name)!!)
```

Comment:
Do we have a way to tell that the device is compatible with the Leia library? 

What I'm thinking is, people who use this app on a regular Android phone would probably never use Leia mode, and people using it on a LumePad would probably only use Leia mode. If that's true, I think it'd make sense to leave the default mode alone, but always automatically use Leia mode when possible.

```suggestion
        videoMode = thisDevice.is(leia)
            ? VideoMode.LEIA
            : VideoMode.valueOf(prefs.getString("video_mode", VideoMode.LEIA.name)!!)
```

Reply:
thats a smart idea. there is a way i believe to query the device to see if it is a leia device. let me look into that and get back to you

Reply:
implemented in next PR